cms/envs/common.py

```diff
@@ -1255,3 +1255,7 @@
 ############## Settings for the Enterprise App ######################
 
 ENTERPRISE_ENROLLMENT_API_URL = LMS_ROOT_URL + "/api/enrollment/v1/"
+
+############## Settings for the Discovery App ######################
+
+COURSE_CATALOG_API_URL = None
```

Comment:
I can see it is already in edx-configurations [HERE](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L832) . right  ?